.github/ISSUE_TEMPLATE/new-dbt-test.md

```diff
@@ -0,0 +1,49 @@
+# New DBT Test
+
+## Issue Summary
+
+_Provide a brief description of the test and its motivation._
+
+## Prerequisites
+
+Before proceeding, ensure you have a basic understanding of DBT tests. The [official DBT documentation on data tests](https://docs.getdbt.com/docs/build/data-tests) is a great resource.
+
+## Checklist
+
+Our workflow focuses on creating reusable generic test templates in the `dbt/tests/generic/` directory, which are invoked in the `schema.yaml` file for a given data model. Follow these steps to add a new test:
+
+1. **Check for Existing Tests:** Ensure the test or a similar functionality does not already exist among our generic templates. Consider modifying an existing template if it closely aligns with the required functionality.
+
+2. **Create Test Template:**
+   - [ ] Add a new test template in the `dbt/tests/generic/` directory.
+   - Example: View an existing DBT test template [here](https://github.com/ccao-data/data-architecture/blob/master/dbt/tests/generic/test_unique_combination_of_columns.sql).
+   - _We use [Jinja](https://jinja.palletsprojects.com/en/3.1.x/templates/) to parameterize the generic test templates._
+
+3. **Integrate Test with Data Model:**
+   - [ ] Include the new test in the `schema.yaml` file under the `dbt/models/` directory for the specific data model.
+   - Example: Here's how the test is implemented for the `default.vw_pin_universe` view:
+
+    ```yaml
+    - unique_combination_of_columns:
+        name: default_vw_pin_universe_unique_by_14_digit_pin_and_year
+        combination_of_columns:
+            - pin
+            - year
+    ```
```

Comment:
**[Suggestion]** There are a couple pieces of key information that are missing here:

* QC tests need either the test or the model to be tagged with a tag that starts with `test_qc_*` (usually `test_qc_iasworld` for an iasworld test) in order to run as part of our QC test suite
* QC tests require a few extra properties in order to be properly displayed in the output workbook:
    * `meta.description` (required): Short description of the test
    * `meta.category` (optional): Category for the test, if the test is not configured with a default category in the [`transform_dbt_test_results`](https://github.com/ccao-data/data-architecture/blob/master/.github/scripts/transform_dbt_test_results.py) script
    * `meta.table_name` (optional): The iasWorld table name that the test operates on, if one cannot be automatically determined from the test

This is a lot of information, and I'm not sure whether it's better documented here in the issue template or elsewhere in our dbt documentation.

Reply:
I added a link out to the docs. You'll have to let me know what the right balance is.